### ISSUE TITLE
Design a simple logo for RatingReviews

### ISSUE THREAD

**Issue by Inconnu08** [Inconnu08/android-ratingreviews#1]
A simple material logo that suits RatingReviews

**Comment by adriancoman** [comment]
Had anything in mind regarding this? Or maybe some inspiration material?

**Comment by Inconnu08** [comment]
Not really. I was thinking something simple and materialistic a huge star with rating bars? 
I'm open for anything. Feel free to share your idea!
